app/assets/javascripts/discourse/app/widgets/quick-access-profile.js

```diff
@@ -6,12 +6,16 @@ import { createWidgetFrom } from "discourse/widgets/widget";
 import showModal from "discourse/lib/show-modal";
 import { dateNode } from "discourse/helpers/node";
 
-const _extraItems = [];
+let _extraItems = [];
 
 export function addQuickAccessProfileItem(item) {
   _extraItems.push(item);
 }
 
+export function resetQuickAccessProfileItems() {
+  _extraItems = [];
```

Comment:
```suggestion
  _extraItems.length = 0;
```

Same as https://github.com/discourse/discourse/pull/21084/files#r1164736644